### PR TITLE
Run Node.js ARM workflow on GitHub runner

### PR DESCRIPTION
**Pull request by louwers**
Work around for gcc bug. https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105651

### CODE REVIEW

File: .github/workflows/node-ci.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ jobs:
         include:
           - runs-on: ubuntu-22.04
             arch: x86_64
-          - runs-on: [self-hosted, linux, ARM64]
+          - runs-on: MapLibre_Native_Linux_ARM_8_core
```
Comment (acalcutt) [inline .github/workflows/node-ci.yml:79]: Don't forget this runner change also needs to be done in node-release

**Comment by acalcutt** [comment]
Is 'MapLibre_Native_Linux_ARM_8_core' a ubuntu 22.04 runner?

**Comment by louwers** [comment]
@acalcutt Ubuntu 24.04 actually https://github.com/maplibre/maplibre/issues/413

**Comment by acalcutt** [comment]
Ubuntu 24.04 would change the compatibility, this would no longer work on ubuntu 22.04 we say we support in the README. I am not sure I am ready to move tot hat but at some point we should.

The linux version is picky about the libs of the system it was build on.

**Comment by birkskyum** [comment]
> Ubuntu 24.04 would change the compatibility, this would no longer work on ubuntu 22.04 we say we support in the README. I am not sure I am ready to move to that but at some point we should.
> 
> The linux version is picky about the libs of the system it was build on.


There is a 22.04 option, so let me know if it would be better to recreate the runner with that instead.
Screenshot 2024-09-22 at 09 09 06

**Comment by acalcutt** [comment]
I think it would be good to do 22.04 for now, at least until we are ready for a new breaking release.

**Comment by louwers** [comment]
@birkskyum Could you make two and include the Ubuntu version in the name?

**Review by cgalvan — APPROVED** [review]

**Comment by acalcutt** [comment]
Any update on this @birkskyum and @louwers ? Can we make a Ubuntu 20.04 for this build for now? It looks like right now the existing runner is already down, so moving it would be good.

File: .github/workflows/node-release.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
         include:
           - runs-on: ubuntu-22.04
             arch: x86_64
-          - runs-on: [self-hosted, linux, ARM64]
+          - runs-on: MapLibre_Native_Linux_ARM_8_core
```
Comment (acalcutt) [inline .github/workflows/node-release.yml:49]: Don't forget to change this to match ci

**Comment by acalcutt** [comment]
I guess I was wrong....we were running on 22.04 before. sorry about that!

File: .github/workflows/node-release.yml (hunk 2 above)
Comment (louwers) [reply .github/workflows/node-release.yml]: CI already complained! :)

**Review by acalcutt — APPROVED** [review]

**Pull request merged**